docs/api/QUIC_SETTINGS.md

```diff
@@ -61,18 +67,24 @@ typedef struct QUIC_SETTINGS {
     uint32_t KeepAliveIntervalMs;
     uint16_t PeerBidiStreamCount;
     uint16_t PeerUnidiStreamCount;
-    uint16_t RetryMemoryLimit;
-    uint16_t LoadBalancingMode;
+    uint16_t RetryMemoryLimit;                  // Global only
+    uint16_t LoadBalancingMode;                 // Global only
     uint8_t MaxOperationsPerDrain;
     uint8_t SendBufferingEnabled            : 1;
     uint8_t PacingEnabled                   : 1;
     uint8_t MigrationEnabled                : 1;
     uint8_t DatagramReceiveEnabled          : 1;
-    uint8_t ServerResumptionLevel           : 2;
+    uint8_t ServerResumptionLevel           : 2;    // QUIC_SERVER_RESUMPTION_LEVEL
     uint8_t VersionNegotiationExtEnabled    : 1;
     uint8_t RESERVED                        : 1;
     const uint32_t* DesiredVersionsList;
     uint32_t DesiredVersionsListLength;
+    uint16_t MinimumMtu;
```

Comment:
I think if you arrange these from largest to smallest, you'll get better packing in the struct.

Reply:
Yeah, but I don't want to change the previous settings.

Reply:
Its a breaking change to rearrange them.

Reply:
Also, these 2 are here because theres 4 bytes of padding after DesiredVersionsListLength.

Reply:
I understand that. I mean among these new settings, you can rearrange them for better packing

Reply:
oh, so it is the docs. My mistake.